repo/content/content_manager.go

```diff
@@ -945,6 +945,9 @@ type ManagerOptions struct {
 	TimeNow                func() time.Time // Time provider
 	DisableInternalLog     bool
 	PermissiveCacheLoading bool
+	// Whether or not writes, for index compaction and cleanup, are permitted
+	// while reading the indexes.
```

Comment:
```suggestion
	// Whether or not writes, for epoch advance, index compaction and cleanup,
	// are permitted while reading the indexes.
```

Reply:
Ended up simplifying the flag wiring. So this comment is no longer needed.